aiohttp/client_exceptions.py

```diff
@@ -238,17 +239,34 @@ class InvalidURL(ClientError, ValueError):
 
     # Derive from ValueError for backward compatibility
 
-    def __init__(self, url: Any) -> None:
+    def __init__(self, url: StrOrURL, description: Optional[str] = None) -> None:
         # The type of url is not yarl.URL because the exception can be raised
         # on URL(url) call
-        super().__init__(url)
+        if description:
+            super().__init__(url, description)
```

Comment:
I think, this could be unconditional.

Reply:
These don't produce the same result, due to the parent class:
`InvalidURL('url', None)` vs `InvalidURL('url')`

Reply:
Can we resolve this conversation?

Reply:
Okay, fair. Marking as resolved.